CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ A [separate changelog is kept for rand_core](rand_core/CHANGELOG.md).
 You may also find the [Upgrade Guide](https://rust-random.github.io/book/update.html) useful.
 
 ## [Unreleased]
+### Changed
+- The dependency on `rand_chacha` has been replaced with a dependency on `chacha20`. This changes the implementation behind `StdRng`, but the output remains the same. There may be some API breakage when using the ChaCha-types directly as these are now the ones in `chacha20` instead of `rand_chacha`.
```

Comment:
I added some equivalence proptests which (in addition to the existing unit tests) ensure the two implementations generate equivalent outputs: https://github.com/RustCrypto/stream-ciphers/pull/435

However, in doing so I did notice `rand_chacha` uses the legacy 64-bit nonce for `set_stream`, which currently isn't supported by `chacha20`. However, it's already a generic parameter `StreamId` which supports a variety of types, so I think it should be fairly straightforward to make it backwards compatible with `u64` too.

Reply:
I'd like to see some form of longer counters before release, yes.

This could also be viewed as a difference between the requirements of a cipher and of a PRNG: one has a testable data length, the other does not.

Reply:
`chacha20` v0.10.0-rc.1 is out with 64-bit counters for all RNGs